src/hardware/vga_crtc.cpp

```diff
@@ -205,24 +205,13 @@ void vga_write_p3d5(io_port_t, io_val_t value, io_width_t)
 				vga.draw.address_line_total *= 2;
 			}
 		} else {
-			auto flipped = new_val;
-			flipped.maximum_scan_line = new_val.maximum_scan_line ^ 1;
-			flipped.start_vertical_blanking_bit9.flip();
-			flipped.is_scan_doubling_enabled.flip();
-
-			// Only true if ALL of the following three bits have
-			// been changed:
-			//   - LSB of `maximum_scan_line`
-			//   - `start_vertical_blanking_bit9`
-			//   - `is_scan_doubling_enabled`
-			// and all other bits are unchanged.
-			//
-			// This revised criteria was taken from DOSBox X. The
-			// old DOSBox criteria was "if ALL bits have been
-			// changed EXCEPT for `line_compare_bit9`"; I can't see
-			// how could that possibly work.
```

Comment:
Oh, that comment... Clearly, I was confused. If you replace "ALL" with "ANY" in my sentence, that's the answer 😏